src/sheet.tsx

```diff
@@ -165,7 +165,10 @@ type SheetProps = {
 
 const Sheet = React.forwardRef<HTMLDivElement, SheetProps>(({onClose, children, dataAttributes}, ref) => {
     const {texts} = useTheme();
-    const [modalState, dispatch] = React.useReducer(modalReducer, 'closed');
+    const [modalState, dispatch] = React.useReducer(
+        modalReducer,
+        process.env.NODE_ENV === 'test' ? 'open' : 'closed'
```

Comment:
This should fix: https://github.com/Telefonica/global-checkout-webview/blob/a5dd8471303747fb9acae3a018fb5804c45507ac/web/utils/test-utils/index.tsx#L189